TM1py/Services/FileService.py

```diff
@@ -21,13 +22,19 @@ def __init__(self, tm1_rest: RestService):
         else:
             self.version_content_path = 'Blobs'
 
-    def get_names(self, **kwargs) -> bytes:
+    def get_names(self, return_list: bool=False, **kwargs) -> bytes:
+        """ return list of blob file names
+
+        :param return_list: True to return list, otherwise will return bytes        
+        """
 
         url = format_url(
             "/Contents('{version_content_path}')/Contents?$select=Name",
             version_content_path=self.version_content_path)
 
-        return self._rest.GET(url, **kwargs).content
+        response = self._rest.GET(url, **kwargs).content
+
+        return response if not return_list else [file['Name'] for file in json.loads(response)['value']]
```

Comment:
TBH I don't think the `get_names` function should return bytes. I'm not sure why it exists.

Maybe we can create a `get_all_names` function instead. 
Similar to what we have in the other services it could return a list of strings